plugins/modules/katello_sync_plan.py

```diff
@@ -122,6 +124,7 @@ def main():
             enabled=dict(type='bool', required=True),
             sync_date=dict(required=True),
             cron_expression=dict(),
+            state=dict(default='present', choices=['present_with_defaults', 'present', 'absent']),
```

Comment:
I like that the only thing needed to support that is the addition to the `state` dict.

Reply:
Yes ! I like this too !